### PR TITLE
comment out default values for the LSF Grid from the origen_site_conf…

### PR DESCRIPTION
**Pull request by priyavadan**
…ig.yml file

### CODE REVIEW

**Comment by pderouen** [comment]
I have no idea what the implications of this change are. But, it seems fine to me. Why are the GitHub Actions not running?

**Comment by priyavadan** [comment]
good question, i do not know why Github Actions did not  run, maybe because the code itself  did not change just a configuration file that isnt used in the spec tests?

based on my conversation with @ginty , these changes should be okay for NXP's use case too @pderouen

**Comment by coreyeng** [comment]
Yes, these should be fine for us. I know 2 of them are even correct anymore and 1 I'm pretty sure isn't, but not 100%. The other is nil and number of cores we usually boost anyway. We have our own site config I can request changes to if need be as well, and it'd only effect Windows users who update or when we invoke globally from this release forward.

TL;DR, its fine.

**Review by coreyeng — APPROVED** [review]

**Comment by priyavadan** [comment]
@pderouen  any concerns?

**Review by pderouen — APPROVED** [review]
No concerns
